common/source/docs/common-rangefinder-landingpage.rst

```diff
@@ -72,7 +72,7 @@ based upon your set-up.
     Maxbotix Analog Sonar <common-rangefinder-maxbotix-analog>
     RPLidar A2 360 degree laser scanner <common-rplidar-a2>
     ST VL53L0X / VL53L1X Lidar <common-vl53l0x-lidar>
-    TeraRanger One/EVO Rangefinders <common-teraranger-one-rangefinder>
```

Comment:
Our existing users may no longer be able to find the documentation for the devices if we remove this.

Perhaps we could add a `.. note:` to the page indicating it is no longer sold instead?